jekyll/_cci2/server-3-whats-new.adoc

```diff
@@ -98,7 +114,6 @@ The new Traefik pos will then start ot schedule correctly.
 
 === Known issues
 
-* KOTS admin configuration incorrectly selects the Nomad mTLS as `enabled` during setup. It should be set to mTLS `disabled` until after nomad clients have been deployed.
```

Comment:
This is still an issue for 3.2.0, just not 3.2.1

Reply:
Yeah this is a problem with the way I have this doc formatted, there's one known issues section for all releases which clearly doesn't work. I'll fix that.

Reply:
Oh I'm talking nonsense, ignore me. I'll fix!